aws.yaml

```diff
@@ -1,4 +1,35 @@
 - active: true
+  apps:
```

Comment:
This is updating the 11.0.0 release but it's already active. Please add a 11.0.1 WIP release with the new cluster-operator release and apps.

This is easy to miss as the current master doesn't have a WIP for 11.X.

Reply:
Should be 11.1.0? since it's not bug fix actually.

Reply:
Let's use patch for now. These changes are about improving our developer experience. This is more interesting for us than for customers.

When this WIP version is released we can upgrade to a minor version if necessary.